modules/dataquery/ajax/queryEqual.php

```diff
@@ -17,13 +17,23 @@
 $fieldName = $_REQUEST['field'];
 $value = $_REQUEST['value'];
```

Comment:
What about `$value = is_numeric($value) ? $value : "\"$value\"";` and remove the if and the escaped quotes?